LICENSE

```diff
@@ -1,201 +0,0 @@
-                                 Apache License
```

Comment:
uhhhhh re-add?

Reply:
the stencil processor used to unset oss: true and i forgot to rebootstrap after fixing